app/models/menu.rb

```diff
@@ -1,7 +1,7 @@
 class Menu < ApplicationRecord
   has_many :menu_items
 
-  enum dining_hall: [ :frank, :frary, :claremont_mckenna, :harvey_mudd, :scripps, :pitzer ]
+  enum dining_hall: [ :frank, :frary, :oldenborg, :claremont_mckenna, :harvey_mudd, :scripps, :pitzer ]
```

Comment:
Tiny change is necessary here. Enums are ordered, which means in our database `:frank => 0`, `:frary => 1`, etc. Adding `:oldenborg` to the middle of this list would actually cause Menu's previously stored as `:claremont_mckenna` (or `2`, within the database) would be errantly read as `:oldenborg` in the future. 

The effect of this change within our menu import system would actually be remedied by the fact that we clear out existing menus each time on import, but it's still a good thing to have in mind when dealing with Enums in the future.